src/js/App/NoAccess.js

```diff
@@ -0,0 +1,65 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+// import { Main, PageHeader, PageHeaderTitle } from '@redhat-cloud-services/frontend-components';
+
+import {
+    Title,
+    Button,
+    EmptyState,
+    EmptyStateVariant,
+    EmptyStateIcon,
+    EmptyStateBody
+} from '@patternfly/react-core';
+
+import { LockIcon } from '@patternfly/react-icons';
+
+const NoAccess = ({ activeAppTitle }) => {
+    return (
+        <Fragment>
+            <section
+                className="pf-m-light pf-c-page-header pf-c-page__main-section pf-m-light"
```

Comment:
Could you just use the page header component here?

Reply:
@ryelo I tried, but it kept giving some sort of router error in the console whenever I imported anything from `frontend-components`. So, had to opt-in for this workaround

Reply:
Works for me!